tests/data_validation/data_validation_core_plugin.test.ts

```diff
@@ -112,7 +112,7 @@ describe("Data validation", () => {
 
       expect(getDataValidationRules(model, sheetId)).toMatchObject([
         { id: "id", ranges: ["A1", "A4:A5"], criterion: { type: "textContains", values: ["1"] } },
-        { id: "id2", ranges: ["C2:2"], criterion: { type: "textContains", values: ["2"] } },
+        { id: "id2", ranges: ["C2:Z2"], criterion: { type: "textContains", values: ["2"] } },
```

Comment:
why? ^^

Reply:
why not 🙃
--> fixed in a new commit to squash